buildenv/jenkins/common/variables-functions

```diff
@@ -363,4 +452,34 @@ def printStackTrace(e) {
     echo e.toString()
 }
 
+/*
+* Sets USER_CREDENTIALS_ID to the value provided in the variables file or an
+* empty string.
+*/
+def set_user_credentials() {
+    USER_CREDENTIALS_ID = get_git_user_credentials_id()
+    if (!USER_CREDENTIALS_ID) {
+         USER_CREDENTIALS_ID = ''
```

Comment:
if `USER_CREDENTIALS_ID` does not get populated from the variable file, what will the value be? Just wondering if its necessary to set it to an empty string.

Reply:
null and it will throw an error if passed to the downstream, thus the empty string.

Reply:
@AdamBrousseau  ready for review, thx